app/services/flags/create_flag.rb

```diff
@@ -26,23 +26,16 @@ class Flags::CreateFlag
 
   private
 
-  def unique_name(name:)
-    !Flag.custom.where(name: name).exists?
+  def invalid_access(guardian:)
+    guardian.can_create_flag?
   end
 
-  def instantiate_flag(name:, description:, applies_to:, require_message:, enabled:)
```

Comment:
Since parameters are validated by a contract, we should not try to access them directly but instead go through the contract.